sim/shaman/runes.go

```diff
@@ -95,18 +121,59 @@ func (shaman *Shaman) applyMaelstromWeapon() {
 	})
 }
 
+const ShamanPowerSurgeProcChance = .05
+
 func (shaman *Shaman) applyPowerSurge() {
 	if !shaman.HasRune(proto.ShamanRune_RuneWaistPowerSurge) {
 		return
 	}
 
+	var affectedSpells []*core.Spell
+
+	procAura := shaman.RegisterAura(core.Aura{
+		Label:    "Power Surge Proc",
+		ActionID: core.ActionID{SpellID: 440285},
+		Duration: time.Second * 10,
+		OnInit: func(aura *core.Aura, sim *core.Simulation) {
+			affectedSpells = core.FilterSlice(
+				core.Flatten([][]*core.Spell{
+					shaman.ChainLightning,
+					shaman.LightningBolt,
+					{shaman.LavaBurst},
```

Comment:
Looks like this should be Chain Lightning, Chain heal, and Lava Burst
<img width="304" alt="image" src="https://github.com/wowsims/sod/assets/12898988/30b14796-c2af-4b7a-b41f-7c698e6b5ff9">